src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json

```diff
@@ -5,8 +5,8 @@
       "Microsoft.NETCore.Console": "1.0.0-rc2-23530",
 
       "coveralls.io": "1.4",
-      "OpenCover": "4.6.166",
-      "ReportGenerator": "2.3.4",
+      "OpenCover": "4.6.261-rc",
```

Comment:
You will need to update the related versions in corefx repo as well once this gets integrated.